<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxCollage">
      <property name="title">
       <string> Коллаж</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menuViewShowWindow">
     <property name="title">
      <string>Показать</string>
     </property>
     <addaction name="actionViewShowWindowImages"/>
     <addaction name="actionViewShowWindowSettings"/>
    </widget>
    <addaction name="menuViewShowWindow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetImages">
   <property name="windowTitle">
    <string>Изображения</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="ImageListWidget" name="imageListWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetSettings">
   <property name="windowTitle">
    <string>Настройки</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionViewShowWindowImages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Изображения</string>
   </property>
  </action>
  <action name="actionViewShowWindowSettings">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageListWidget</class>
   <extends>QWidget</extends>
   <header>ImageList/ImageListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
